lib/Serialization/Serialization.cpp

```diff
@@ -898,6 +898,8 @@ void Serializer::writeBlockInfoBlock() {
                               decls_block::GENERIC_REQUIREMENT);
   BLOCK_RECORD_WITH_NAMESPACE(sil_block,
                               decls_block::LAYOUT_REQUIREMENT);
+  BLOCK_RECORD_WITH_NAMESPACE(sil_block,
+                              decls_block::INLINABLE_BODY_TEXT);
```

Comment:
This one won't actually ever appear in the SIL namespace, so you don't need to add it here.